### DISCUSSION TITLE
How to run with libPcmMsr.dylib in current folder?

### DISCUSSION THREAD

**Discussion by roddy20** [opcm/pcm#279] [Q&A] [answered]
Hello! it s not a problem, but a question
need to run it without installing libPcmMsr.dylib to system
now it works:
```
cd "$(dirname "$0")"
sudo kextload PcmMsrDriver.kext
export DYLD_LIBRARY_PATH=.:$DYLD_LIBRARY_PATH
MSR_IA32_PERF_STATUS=$(./pcm-msr.x 0x198 | grep value | awk '{print $3}')
```
maybe there is a better way?
Thanks

**Comment by opcm** [comment]
looks good to me

**Comment by opcm** [comment]
question answered

**Comment by roddy20** [comment, marked as answer]
doesnot work anymore (since one of the newest commits)
found another option
install_name_tool -change /usr/lib/libPcmMsr.dylib @executable_path/libPcmMsr.dylib pcm-msr.x

**Comment by opcm** [comment]
Not sure why. Thanks for the workaround. Moving to discussion to make it more visible

**Reply by roddy20** [reply]
Thanks!
I think it is not very difficult to do the same adding some keys for "make" during compilation, but I am not an advanced programmer
